### DISCUSSION TITLE
Overwrites the master branch

### DISCUSSION THREAD

**Discussion by bennihtm** [upptime/upptime#68] [Q&A] [answered]
The GitHub Action overwrites the master branch and therefor can't run ever again.

**Comment by AnandChowdhary** [comment]
I don't understand, can you please elaborate?

**Comment by bennihtm** [comment]
I ran the Setup CI Action and it ran fine, but after that the master branch had been overwritten with the content of the gh-pages branch.

**Comment by AnandChowdhary** [comment]
Can you share a link to your repository with me? This should not be the case since we explicitly mention the `gh-pages` branch.

**Comment by bennihtm** [comment]
Here it is: https://github.com/awesomebible/awesomebible.github.io

**Comment by AnandChowdhary** [comment, marked as answer]
Aah. This is a special case because USERNAME.github.io is a special repository name, where it's published not from the gh-pages branch but the default branch.

You should create a new Upptime repo with a different name.

**Comment by bennihtm** [comment]
Okay, thanks!

**Comment by AnandChowdhary** [comment]
I'll close this for now, but please feel free to re-open this if this doesn't solve your issue! 😄
